implementations/elixir/ockam/ockam_kafka/test/interceptor/outlet_manager_test.exs

```diff
@@ -67,29 +67,29 @@ defmodule Ockam.Kafka.Interceptor.OutletManager.Test do
     ssl = false
     ssl_options = []
 
-    base_port = 7000
+    base_port = 11_000
     allowed_ports = 10
     base_route = []
 
     start_supervised!({InletManager, [base_port, allowed_ports, base_route, outlet_prefix]})
     start_supervised!({OutletManager, [outlet_prefix, ssl, ssl_options]})
 
-    Ockam.Transport.TCP.start(listen: [port: 4000])
+    Ockam.Transport.TCP.start(listen: [port: 12_000])
```

Comment:
not for now but if the problem is the port collision,  we could make the  port optional and let the OS choose one for us, returning it from the start() function? .  So the rest of the test can refer to it.    Seems useful only for tests but could be an easy win?

Reply:
For this one we can do that, for inlet-outlet manager test we set up port range, which is more complex.
But this change is a quick-fix so we don't get the flaky tests so often.